thefuck/rules/switch_lang.py

```diff
@@ -63,15 +77,43 @@ def _switch_command(command, layout):
     return ''.join(_switch(ch, layout) for ch in command.script)
 
 
+def _decompose_korean(command):
+    def _change_double(ch):
+        if ch in DOUBLE_LIST:
+            return DOUBLE_MOD_LIST[DOUBLE_LIST.index(ch)]
+        else:
+            return ch
```

Comment:
Nit: this `else` statement is unnecessary, as its `if` counterpart has a `return` in its block (on line 83). So it can be removed and the `return ch` statement can be stepped one indentation back. That's for improved readability.

```suggestion
        return ch
```